projects/DensePose/doc/TOOL_APPLY_NET.md

```diff
@@ -35,6 +35,67 @@ python apply_net.py dump configs/densepose_rcnn_R_50_FPN_s1x.yaml DensePose_ResN
 ```
 
 
+
+If you want to load the pickle file of DensePose model results: 
+```
+f = open('/your_result_path/result.pkl', 'rb')
+data = pickle.load(f)
+```
+
+
+The pickle loading code should be run in the `/your_detectron2_path/detectron2_repo/projects/DensePose/` directory, 
+or `ModuleNotFoundError: No module named 'densepose'` will occur. 
+
+If you have to run the code in other directory, simply add this before pickle loading:
+```
+import sys
+sys.path.append("/your_detectron2_path/detectron2_repo/projects/DensePose/")
+```
+
+The file `result.pkl` contain the list of results per image, for each image the result is a dictionary:
+```
+data: [{'file_name': '/your_path/image1.jpg', 
+        'scores': tensor([0.9884]), 
+        'pred_boxes_XYXY': tensor([[ 69.6114,   0.0000, 706.9797, 706.0000]]), 
```

Comment:
I don't feel this is correct because the string "pred_boxes_XYXY" does not appear in the codebase.
It's probably "pred_boxes".

Reply:
I have checked the code in line 148-155 of `apply_net.py`:
```
    if outputs.has("pred_boxes"):
        result["pred_boxes_XYXY"] = outputs.get("pred_boxes").tensor.cpu()
        if outputs.has("pred_densepose"):
            boxes_XYWH = BoxMode.convert(
                result["pred_boxes_XYXY"], BoxMode.XYXY_ABS, BoxMode.XYWH_ABS
            )
            result["pred_densepose"] = outputs.get("pred_densepose").to_result(boxes_XYWH)
    context["results"].append(result)
```
It seems that the dump action of `apply_net.py` uses the string "pred_boxes_XYXY" as the key to store the bounding box array, maybe this typo should be further fixed?

Reply:
I see. I didn't know there is an inconsistency in the naming. Then let's keep it the old way to be safe.

Reply:
Got it.